presto-cli/src/main/java/com/facebook/presto/cli/Query.java

```diff
@@ -143,6 +143,7 @@ else if (results.getColumns() == null) {
         if (statusPrinter != null) {
             statusPrinter.printFinalInfo();
         }
+        printWarnings(errorChannel);
```

Comment:
Please format the commit msg ("Warnings: Legacy order by behavior and propagation") according to the convention (don't have a link at hand, but should be imperative form e.g. "Print warnings when ..")